mmv1/third_party/terraform/tests/resource_active_directory_domain_update_test.go

```diff
@@ -78,3 +83,36 @@ func testAccADDomainUpdate(context map[string]interface{}) string {
 	`, context)
 
 }
+
+func testAccCheckActiveDirectoryDomainDestroyProducer(t *testing.T) func(s *terraform.State) error {
```

Comment:
why is this needed?

Reply:
We used to have this function generated by mmv1 but the generated test file got removed since we skip all the tests. I just copied it from the previous generated file, otherwise the build is going to fail as we have the handwritten test `TestAccActiveDirectoryDomain_update` calling this function (even though it is also skipped)